features/cli/deploy.feature

```diff
@@ -377,8 +386,16 @@ Feature: deployment related features
       | name     | ab-example-a     |
       | replicas | 1                |
     Then the step should succeed
-    Given I wait until number of replicas match "0" for replicationController "ab-example-b-1"
-    Given I wait until number of replicas match "1" for replicationController "ab-example-a-1"
+    Given number of replicas of "ab-example-b" deployment config becomes:
+      | desired   | 0 |
+      | current   | 0 |
+      | updated   | 0 |
+      | available | 0 |
+    Given number of replicas of "ab-example-a" deployment config becomes:
+      | desired   | 1 |
+      | current   | 1 |
+      | updated   | 1 |
+      | available | 1 |
```

Comment:
Do we need line no: 389 to 393 ? I understand that we are waiting for replicas to become 1.